lib/dragonfly_libvips/processors/thumb.rb

```diff
@@ -26,15 +26,19 @@ def call(content, geometry, options = {})
             cmyk:  img.get('interpretation') == :cmyk,
             jpeg:  content.mime_type == 'image/jpeg'
           )
+
           case process
             when :shrink
               thumbnail_options.except!(:height, :size)
               thumbnail_options[:xshrink] = dimensions.x_scale
               thumbnail_options[:yshrink] = dimensions.y_scale
               img.shrink(dimensions.x_scale, dimensions.y_scale, **thumbnail_options)
             when :crop
-              thumbnail_options.except!(:height, :size, :no_rotate)
-              img.crop(dimensions.x, dimensions.y, dimensions.width, dimensions.height, **thumbnail_options)
+              if dimensions.resize_width
+                img = img.thumbnail_image(dimensions.resize_width.ceil, size: :both, height: dimensions.resize_height)
+              end
+
+              img.crop(dimensions.x, dimensions.y, dimensions.width, dimensions.height)
```

Comment:
`crop` does not support the `scale` parameter. We need to add an extra resize step in case we're using a geometry string.